web/test/pages/onboarding/onboarding-additional-business.test.tsx

```diff
@@ -26,6 +27,7 @@ jest.mock("@/lib/api-client/apiClient", () => ({
 
 const mockApi = api as jest.Mocked<typeof api>;
 const Config = getMergedConfig();
+const { none } = Config.profileDefaults.fields.foreignBusinessTypeIds.default.optionContent;
```

Comment:
[sand] I'd love to see this variable assigned to something more meaningful in this context, akin to the following:

```typescript
const { none: nexusNoneOfTheAboveCheckboxLabel } = Config.profileDefaults.fields.foreignBusinessTypeIds.default.optionContent;
```

(Obviously, open to better names here.)